src/components/OrderSummary/OrderSummary.js

```diff
@@ -25,20 +25,24 @@ export class OrderSummaryHeader extends Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    id: PropTypes.string,
     title: PropTypes.string,
   };
 
   static defaultProps = {
+    id: 'order__summary__header__id',
```

Comment:
Given hard-coded ID tends to cause ID duplication (though you made it configurable), I’d suggest making it random, e.g. `order__summary__header__id_${Math.random().toString(36).substr(2)}`. Same for `order__summary__footer__id`.